### DISCUSSION TITLE
Can i develop a Addon by myself?

### DISCUSSION THREAD

**Discussion by Eloas6472** [ScratchAddons/ScratchAddons#5799] [Help]
Can i develop a Addon by myself or help making the Extension better? I'm a Developer and have many Experience at developing. And i'm loving Scratch Addons and want to help Developing. Is there a way to help at the Extension or Develope a Addon by myself?

Thanks,
Eloas

**Comment by WorldLanguages** [comment]
See https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CONTRIBUTING.md

**Comment by DNin01** [comment]
Welcome to GitHub! To make an addon, you'll need to know the basics of contributing and committing on GitHub. If you'd like to use the GitHub website to do that, our [contributing guidelines](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CONTRIBUTING.md) has the list of steps you'll need. Or you can use [Git](https://git-scm.com/) or [GitHub Desktop](https://desktop.github.com/).

**Comment by iqnite** [comment]
Of course you can! Learn how at https://scratchaddons.com/docs/develop/
